timeago/src/main/java/com/github/kinnonii/timeago/TimeAgo.java

```diff
@@ -474,18 +469,10 @@ public String getCountryCode(){
 	 * @param locale The new language. Must be equals to 'en', 'es', 'it', 'fr' or 'de'.
 	 */
 	 public void changeLanguage(String locale){
-	 	if(locale.equals("en"))
-			initLanguage("EN");
-		else if(locale.equals("es"))
-			initLanguage("ES");
-		else if(locale.equals("it"))
-			initLanguage("IT");
-		else if(locale.equals("fr"))
-			initLanguage("FR");
-		else if(locale.equals("de"))
-			initLanguage("DE");
-		else
-			throw new IllegalArgumentException("Locale not recognised or it isn't supported yet.");
-		
+		 if (Arrays.asList(SUPPORTED_LOCALES).contains(locale)) {
```

Comment:
ditto for guard statement... maybe pull this out into a 'commonInit' method or 'changeLanguage' method or something as well since it's duplicated?